packages/playground/src/components/node_selector/TfNodeDetailsCard.vue

```diff
@@ -45,6 +50,13 @@
           </VChip>
         </template>
       </VTooltip>
+      <VTooltip text="Node Country" v-if="node && node.location.country" location="left">
```

Comment:
The county name is already displayed in a chip, why to show it again in the tooltip ?

Reply:
@ehab-hassan requested the design. Would you like to remove it?